docs/Overview.md

```diff
@@ -72,23 +73,24 @@ The packaging projects exist to gather assemblies into their shipping vehicles:
 Project | Responsibility
 --- | ---
 MSI | Builds the MSI file used by most users.
-ApplicationInsights-CI | Builds the NuGet package that will be used by users who wish to scan via automation.
+AxeWindows-CI | Builds the NuGet package that will be used by users who wish to scan via automation.
```

Comment:
>scan via automation [](start = 80, length = 19)

When the package comes from a second repo, the package will cover more than just automation. Not sure what that means for this comment